torch/csrc/jit/tracer.cpp

```diff
@@ -641,6 +649,18 @@ void addOutput(Node* node, const std::vector<at::Tensor>& outputs) {
   }
 }
 
+void addOutput(Node* node, const c10::List<at::Tensor>& outputs) {
```

Comment:
there's a function `addOutput` that already takes `std::vector<at::Tensor>& outputs`, you should just use that. why we need to add a new case here?

Reply:
they have slightly different interfaces from what I understand

Reply:
The list different between c10::List and std::vector is not ideal. @zdevito is working on removing the list specialization in https://github.com/pytorch/pytorch/pull/30734, which hopefully can allow the tracer takes `std::vector<T>` as a constructor of `c10::List`. In that case we should only have one `addOutput` here which takes c10::List<at::Tensor> only.

Reply:
@wanchaol thanks for letting me know. What should we do here? Wait for #30734 to get merged or proceed as is for now?

Reply:
I think we can just merge, @zdevito will do a simple rebase fix after we merged.